ui/components/app/step-progress-bar/step-progress-bar.js

```diff
@@ -19,10 +19,10 @@ export const twoStepStages = {
   PASSWORD_CREATE: 2,
 };
 
-export function ThreeStepProgressBar({ stage }) {
+export function ThreeStepProgressBar({ stage, ...props }) {
   const t = useI18nContext();
   return (
-    <Box>
+    <Box {...props}>
```

Comment:
@adonesky1 I added this but not sure we want to promote this pattern of allowing any props. I know how you could extend props in typescript but not in normal react? Would it be safer to outline the specifc props we are going to use with this component? In this case `marginBottom`? Or could we do something like:

```
ThreeStepProgressBar.propTypes = {
  stage: PropTypes.number,
  ...Box.propTypes,
};

TwoStepProgressBar.propTypes = {
  stage: PropTypes.number,
  ...Box.propTypes,
};
```

I guess this still doesn't solve the being able to pass any props to this component. Would be great to figure out because I can see this being very useful for the redesign components.